Core/src/org/sleuthkit/autopsy/casemodule/UnpackagePortableCaseProgressDialog.java

```diff
@@ -142,65 +149,89 @@ public void propertyChange(PropertyChangeEvent evt) {
         @NbBundle.Messages({
             "UnpackageWorker.doInBackground.errorFinding7zip=Could not locate 7-Zip executable",
             "UnpackageWorker.doInBackground.errorCompressingCase=Error unpackaging case",
-            "UnpackageWorker.doInBackground.canceled=Unpackaging canceled by user",})
+            "UnpackageWorker.doInBackground.canceled=Unpackaging canceled by user",
+            "UnpackageWorker.doInBackground.previousSeenCase=Case {0} has been previously opened do you want to open it again?"})
```

Comment:
Could we change this to specify that a portable case by the same name has already been extracted? We don't know if we've opened the requested case unless we examined its contents.

Reply:
Changed to "Case with name {0} has been previously opened do you want to open it again?"